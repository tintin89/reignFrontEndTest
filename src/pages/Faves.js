import React from 'react'
import Results from '../components/Results/Results';

function Faves() {
    return (
       <Results myFavs/>
    )
}

export default Faves
