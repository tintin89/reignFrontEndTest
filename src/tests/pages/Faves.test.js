import React from 'react';
import  { shallow }  from "enzyme";
import { MemoryRouter } from "react-router-dom";
import Faves from '../../pages/Faves';
import {Provider} from 'react-redux';
import configureStore from 'redux-mock-store';

const mockStore = configureStore();

describe('tests in  <Faves/>',()=>{   
    const initState = {
        favorites: [],        
    };
    let store = mockStore(initState);

    test('Must show correctly the component',()=>{
       
        const wrapper = shallow(
            <Provider store={store}>
             <Faves/>
            </Provider>
        );
        expect(wrapper).toMatchSnapshot()
    })

   
})