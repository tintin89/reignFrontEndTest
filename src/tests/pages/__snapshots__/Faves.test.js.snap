// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests in  <Faves/> Must show correctly the component 1`] = `
<div
  className="results"
/>
`;
